aip/auth/4115.md

```diff
@@ -51,7 +51,7 @@ is called the mTLS configuration endpoint.
 
 The URL of the MDS's mTLS configuration endpoint is: 
 ```
-http://metadata.google.internal/computeMetadata/v1/googleAutoMtlsConfiguration
+http://169.254.169.254/computeMetadata/v1/instance/platform-security/auto-mtls-configuration
```

Comment:
Can you provide context for moving away from using the DNS endpoint?

Reply:
The dns string is google specific, and we wanted to be more generic for tpc use as well. e.g. client libraries running in difference universe could get different dns names.

Reply:
@TimurSadykov Is there any precedence here?